docs/source/large_data.rst

```diff
@@ -0,0 +1,62 @@
+.. module:: statsmodels.base.distributed_estimation
+.. currentmodule:: statsmodels.base.distributed_estimation
+
+Working with "large"-ish data
+=============================
+
+    Big data is like teenage sex: everyone talks about it, nobody really knows
```

Comment:
I think we don't want to have this analogy in our documentation

Reply:
I love that quote... but fair enough. I sanitized it.